wntr/epanet/toolkit.py

```diff
@@ -48,41 +46,15 @@ def ENgetwarning(code, sec=-1):
         sec -= hours * 3600
         mm = int(sec / 60.0)
         sec -= mm * 60
-        header = "At %3d:%.2d:%.2d, " % (hours, mm, sec)
+        header = "%3d:%.2d:%.2d" % (hours, mm, sec)
     else:
-        header = ""
-    if code == 1:
-        return (
-                header
-                + "System hydraulically unbalanced - convergence to a hydraulic solution was not achieved in the allowed number of trials"
-        )
-    elif code == 2:
-        return (
-                header
-                + "System may be hydraulically unstable - hydraulic convergence was only achieved after the status of all links was held fixed"
-        )
-    elif code == 3:
-        return (
-                header
-                + "System disconnected - one or more nodes with positive demands were disconnected for all supply sources"
-        )
-    elif code == 4:
-        return (
-                header
-                + "Pumps cannot deliver enough flow or head - one or more pumps were forced to either shut down (due to insufficient head) or operate beyond the maximum rated flow"
-        )
-    elif code == 5:
-        return (
-                header
-                + "Vavles cannot deliver enough flow - one or more flow control valves could not deliver the required flow even when fully open"
-        )
-    elif code == 6:
-        return (
-                header
-                + "System has negative pressures - negative pressures occurred at one or more junctions with positive demand"
-        )
+        header = "{}".format(code)
+    if code < 100:
+        msg = EN_ERROR_CODES.get(code, 'Unknown warning %s')
```

Comment:
Why are these unknown warnings? Error codes 1-6 are listed as simulation warnings.

Reply:
These are not unknown warnings, the ``get`` argument is passing a default, which means if someone passes 17, for example, we don't know what warning 17 is. If it is 1-6, then it will give the actual warning, not just the "unknown warning" piece.